tests/test_api_auth.py

```diff
@@ -370,11 +349,7 @@ def test_02_auth_without_split(self):
                 self.assertEqual(4031, result['error']['code'], result)
                 self.assertEqual('Authentication failure. Wrong credentials',
                                  result['error']['message'], result)
-            # the realm will be split from the login name
-            if six.PY2:
-                expected = "The user User(login=u'selfservice@realm1', " \
-                           "realm=u'realm3', resolver='') exists in NO resolver."
-            else:
+                # the realm will be split from the login name
                 expected = "The user User(login='selfservice@realm1', " \
```

Comment:
here also

Reply:
fixed in a1e752d6c5abb37fab930e65b3c64b3680e3f3f1.